lib/rpmsg/rpmsg.c

```diff
@@ -176,18 +176,14 @@ struct rpmsg_endpoint *rpmsg_get_endpoint(struct rpmsg_device *rdev,
 
 static void rpmsg_unregister_endpoint(struct rpmsg_endpoint *ept)
 {
-	struct rpmsg_device *rdev;
-
-	if (!ept)
-		return;
-
-	rdev = ept->rdev;
+	struct rpmsg_device *rdev = ept->rdev;
```

Comment:
not related to the commit
Please put this in https://github.com/OpenAMP/open-amp/pull/204/commits/fe6412ba6a0cd6237b7d0ae16e6616d9fbbc5c1a and rename the commit in something like rpmsg_unregister_endpoint management cleanup

Reply:
Done.